src/include/rados.h

```diff
@@ -463,6 +463,8 @@ enum {
 	CEPH_OSD_OP_FLAG_FADVISE_WILLNEED   = 0x10,/* data will be accessed in the near future */
 	CEPH_OSD_OP_FLAG_FADVISE_DONTNEED   = 0x20,/* data will not be accessed in the near future */
 	CEPH_OSD_OP_FLAG_FADVISE_NOCACHE   = 0x40, /* data will be accessed only once by this client */
+	CEPH_OSD_OP_FLAG_WITH_REFERENCE   = 0x80, /* need reference couting */
```

Comment:
instead of just adding this one line from 12feefc10ad88933b3550d5e9ab1bed3b3daceaa to the `enum`, would it make sense to cherry-pick the entire commit in?

Reply:
agreed. i don't think we need `CEPH_OSD_OP_FLAG_WITH_REFERENCE` introduced by #19935 is a dependency of this bug fix.

Reply:
agree